Dockerfile

```diff
@@ -0,0 +1,20 @@
+# pull official base image
+FROM node:14.16.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY yarn.lock ./
+RUN yarn
+# RUN yarn add react-scripts@3.4.1 -g --silent
+
+# add app
+COPY . ./
+
+# start app
+CMD ["yarn", "start"]
```

Comment:
Would it be better to run yarn build here and serving?

Reply:
This is for development only right now. I haven't added a production container setup yet. If we run `yarn build` I don't think we'd get hot-reload

Reply:
Ohhh thats truee